### DISCUSSION TITLE
No values in: using_reservoir_command_areas_totalend on date: 01/01/2013

### DISCUSSION THREAD

**Discussion by luanshilaogao** [iiasa/CWatM#90] [Q&A] [answered]
Step         Date   Discharge
731    31/12/2012   10555.09
========================== CWATM Warning =============================
No values in: using_reservoir_command_areas_totalend on date: 01/01/2013
Could not write: E:\cwat_new\CWatM-output\using_reservoir_command_areas_totalend.nc
732    01/01/2013   10561.02
Start: 12:19:22.921895
End:   12:21:27.916092

Hello, thank you very much for solving my problem.
I am a Chinese student who encountered the above problem while running sample data on the model. How can I solve it? Did I lose any data?

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag, it seems that CWatM is running properly but producing a warning about an output.

In the settings file, is _using_reservoir_command_areas_ one of the output variables, for example, 
**OUT_MAP_TotalEnd = using_reservoir_command_areas**?

You can simply remove _using_reservoir_command_areas_ from the outputs list if you are not using it. Did you include this variable in the outputs?

**Reply by luanshilaogao** [reply]
Thanks so much!
I've solved my problem which I just removed all my outputs variables.
